scripts/singularity_cmd

```diff
@@ -51,7 +51,8 @@ function pass_git_config() {
 
 }
 
-thisdir=$(dirname "$0"| xargs readlink -f)
+thisfile=$(python3 -c 'import sys, os ; print(os.path.realpath(sys.argv[1]))' "$0")
```

Comment:
Ha ha, clever move! But I don't think we should rely (solely) on python to be available

Reply:
That's actually about as good as you're going to get for OS X.  The alternative is to piece together pwd and $0 and then readlink every component in the path -- and then piece those together since readlink might be relative and might be absolute.

python3 -> python might improve portability at the moment.

Reply:
What I meant - use readlink way if it is available and use python way if python available. Yes, better just python not python3